chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -560,6 +593,33 @@ impl PeerManagerActor {
         );
     }
 
+    fn bootstrap_outbound_from_recent_connections(&self, ctx: &mut actix::Context<Self>) {
+        let recent_connections = self.state.peer_store.get_recent_connections();
+
+        for peer_id in recent_connections {
+            let peer_info = PeerInfo { id: peer_id, addr: None, account_id: None };
```

Comment:
I suppose this part is not implemented yet, right? Recent connections should contain address of the peer, right?